packages/vx-grid/src/grids/GridColumns.tsx

```diff
@@ -47,13 +58,18 @@ export default function GridColumns<ScaleInput>({
           x,
           y: height,
         });
+
+        const columnLineProps = gridColumnLineProps(d, i);
+        const lineStroke = columnLineProps.stroke || stroke;
```

Comment:
currently you don't use any other line props that might be specified by `gridLineProps()`.

what if instead of extracting `lineStroke` and `lineWidth`, you keep them as they were and then simply spread the output of `gridLineProps` which could override `stroke/strokeWidth` or not: `<Line stroke={stroke} ... {...gridColumnLineProps(d, i, ticks)} />`